tests/fsharpqa/Source/Conformance/TypesAndTypeConstraints/TypeParameterDefinitions/UnitSpecialization.fs

```diff
@@ -0,0 +1,9 @@
+// #UnitGenericAbstractType 
+//<Expects status=success></Expects>
```

Comment:
it's not an error (this sintax is supported), but can you pls write that as xml like (supported too)? 

```
//<Expects status="success"></Expects>
```

i am trying to convert the suite to nunit, and to simplify i'll need to convert that as xml like (easier to parse, validate)